packages/typespec-ts/test/integration/generated/sharedRoute/src/clientDefinitions.ts

```diff
@@ -59,7 +65,7 @@ export interface ProcessInt {
 
 export interface Routes {
   /** Resource for '/sharedroute/query' has methods for the following verbs: post */
-  (path: "/sharedroute/query"): ListByResourceGroup;
+  (path: "/sharedroute/query"): ListBySubscription;
```

Comment:
what is causing the change here ? operation order change?

Reply:
no the bug is introduced by overload.